ggml/src/ggml-vulkan/CMakeLists.txt

```diff
@@ -69,11 +69,15 @@ if (Vulkan_FOUND)
 
     file(GLOB _ggml_vk_shader_deps "${_ggml_vk_input_dir}/*.comp")
 
+    if (CMAKE_SYSTEM_NAME STREQUAL CMAKE_HOST_SYSTEM_NAME)
```

Comment:
I don't know which is more reliable, but this could also be `if (NOT CMAKE_CROSSCOMPILING)`.